pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Common Issues Using Python on Windows
+description: A list of common issues with Python on Windows and how to fix them
+icon: fab fa-windows
+toc: 3
+---
+
+### When I run `python` in the terminal I get no result, or the Windows Store opens!
+
+By default Windows has an alias for `python` in the terminal to guide you to install it from the [Microsoft
+Store](https://apps.microsoft.com/store/search/python). We don't recommend installing Python from the Microsoft
+Store, see why [here](../microsoft-store). To disable this alias, search "App execution aliases" in Windows search
+and click on "Manage app execution aliases". In the list you should see two options with title "App Installer" and
+descriptions `python.exe` and `python3.exe`. Disable both of them.
```

Comment:
I wrote this, but I sort of wonder whether it would be best not to suggest this. Disabling the alias doesn't really help with anything, as having python on path will overwrite it which is the intended behaviour. 

All this means is that if the user *does* try and use the microsoft store version it wont work.

Reply:
I see, I didn't realise it was a normal path entry.

I also just realised that if you install python from the ms store it adds the other entries (pip, etc) as app execution aliases (there are loads...)
![image](https://user-images.githubusercontent.com/22353562/201529405-96756802-e0f9-4c87-8414-fa7741fbfb26.png)
![image](https://user-images.githubusercontent.com/22353562/201529431-5c23f048-b01e-4ff1-9ff8-50324919d75a.png)
![image](https://user-images.githubusercontent.com/22353562/201529454-350a3c8e-181f-456c-b434-edd0da356cd0.png)

Are the app installer ones the two that are there by default?

Reply:
Yeah (at least on Win 10) when you have the App Installer ones on but no MS store Python installed a python.exe and python3.exe still show up in `C:\Users\user\AppData\Local\Microsoft\WindowsApps`.